pkg/config/default_windows.go

```diff
@@ -36,7 +43,10 @@ func getLibpodTmpDir() string {
 
 // getDefaultMachineVolumes returns default mounted volumes (possibly with env vars, which will be expanded)
 func getDefaultMachineVolumes() []string {
-	return []string{}
+	hd := homedir.Get()
+	vol := filepath.VolumeName(hd)
+	hostMnt := filepath.ToSlash(strings.Replace(hd, vol, "", 1))
```

Comment:
TrimPrefix() may be better than Replace() given the drive is always at the start.